src/sonic-config-engine/config_samples.py

```diff
@@ -242,6 +246,7 @@ def generate_l2_config(data):
             # Ports in use should be admin up, unused ports default to admin down
             if port in downlinks or port in uplinks:
                 data['PORT'][port].setdefault('admin_status', 'up')
+                data['PORT'][port].setdefault('speed', '50000')
```

Comment:
>50000 [](http://example.com/codeflow?start=56&length=5)

Not all the ports on all the platforms support 50G speed. It it the responsibility of port_config.ini or hwsku.json to define the init port speed.